packages/module/src/pipelines/components/nodes/TaskNode.tsx

```diff
@@ -121,7 +121,16 @@ const TaskNode: React.FC<TaskNodeProps & { innerRef: React.Ref<SVGGElement> }> =
   showStatusState = true,
   scaleNode,
   hideDetailsAtMedium,
-  hiddenDetailsShownStatuses = [RunStatus.Failed, RunStatus.FailedToStart, RunStatus.Cancelled],
+  hiddenDetailsShownStatuses = [
+    RunStatus.Failed,
+    RunStatus.FailedToStart,
+    RunStatus.Cancelled,
+    RunStatus.Running,
+    RunStatus.Skipped,
+    RunStatus.Succeeded,
+    RunStatus.Pending,
+    RunStatus.InProgress
+  ],
```

Comment:
By default we only want highlight the failed and cancelled statuses when details are hidden. Apps that want to show others can add them but we should keep the default setting as it was.

Reply:
Reverted this change.